src/__tests__/parse.spec.ts

```diff
@@ -13,112 +10,4 @@ describe('parse', () => {
 
     expect(parse('**simple**')).toEqual(parse('**simple**'));
   });
-
-  describe('jira blocks', () => {
```

Comment:
did you actually verify whether huge jira blocks do cause parsing to stall?

Reply:
On it - there is indeed a perf issue w one particular article. Will fix.